File: strategy/lua/marvin.bak/test/move/defend/armada.lua
local Armada = Class("Test.Move.Defend.Armada", require "group/move/armada")

local G = (require "../base/world").Geometry

Armada.TEST_BALL_START_RECTS = {
		{Vector(G.FieldWidthHalf / 2,4*G.FieldHeightHalf / 5), Vector(G.FieldWidthHalf, G.FieldHeightHalf)},
		{Vector(-G.FieldWidthHalf/2, 4*G.FieldHeightHalf / 5), Vector(-G.FieldWidthHalf, G.FieldHeightHalf)},
--		{Vector(), Vector()}
}

return Armada

File: strategy/typescript/glados/test/move/defend/armada.ts
let Armada = Class("Test.Move.Defend.Armada", require "group/move/armada")

let G = (require "+/base/world").Geometry

Armada.TEST_BALL_START_RECTS = {
		{new Vector(G.FieldWidthHalf / 2,4*G.FieldHeightHalf / 5), new Vector(G.FieldWidthHalf, G.FieldHeightHalf)},
		{new Vector(-G.FieldWidthHalf/2, 4*G.FieldHeightHalf / 5), new Vector(-G.FieldWidthHalf, G.FieldHeightHalf)},
//		{new Vector(), new Vector()}
}

return Armada
